scripts/config/lib/nginx_config_util.rb

```diff
@@ -44,4 +44,16 @@ def self.match_redirects(redirects, uri)
 
     false
   end
+
+  def self.interpolate(string, vars)
+    regex = /\${(\w*?)}/
+
+    string.scan(regex).inject(string) do |acc, capture|
+      var_name = capture.first
+      value = vars[var_name] if vars
+      acc.sub!("${#{var_name}}", value) if value
```

Comment:
Eh. I'm fine with it as is, then. I had been contemplating something like

```ruby
acc.sub!("${#{Regexp.escape var_name}}", value) if value
```

in case of `var_name` itself containing "interesting" characters; I know that's impossible right now (`\w` doesn't include any interesting characters) so I'm happy with it as is. Just being finnicky about edge cases and what-ifs.